src/sentry/tasks/symbolication.py

```diff
@@ -165,18 +195,22 @@ def _do_symbolicate_event(
             return
 
     def _continue_to_process_event(was_killswitched: bool = False) -> None:
-        # After JS processing, we check `get_native_symbolication_function`,
-        # because maybe we need to feed it to another round of
-        # `symbolicate_event`, but for *native* that time.
-        if not was_killswitched and task_kind.platform == SymbolicatorPlatform.js:
-            symbolication_function = get_native_symbolication_function(data)
-            if symbolication_function:
+        # Go through the remaining symbolication platforms
+        # and submit the next one.
+        if not was_killswitched:
+            while symbolicate_platforms:
+                next_platform = symbolicate_platforms.pop(0)
+                # Guard against submitting the same platform twice
+                if next_platform == task_kind.platform:
+                    continue
```

Comment:
I believe with the above update to the defaulting of `symbolicate_platforms`, this won’t be needed anymore.